.github/actions/latest-wrangler/main.py

```diff
@@ -35,12 +35,11 @@
             sys.exit(0)
 
     # TODO: verify package meta is "correct"
-    # https://github.com/dbt-labs/dbt-core/issues/4640 
+    # https://github.com/dbt-labs/dbt-core/issues/4640
 
     # map versions and tags
     version_tag_map = {
-        version["id"]: version["metadata"]["container"]["tags"]
-        for version in package_meta
+        version["id"]: version["metadata"]["container"]["tags"] for version in package_meta
```

Comment:
This is what I was talking about when it reformats my comprehensions.  In this case it does probably improve readability, but in more complex comprehensions having line breaks make logical separation of what's happening can be useful.. 

That said, if my comprehensions are complex enough to require visual clues about their processing, I should probably not use a comprehension.

:)